dangerzone/gui/updater.py

```diff
@@ -28,11 +29,12 @@
 # TODO: Create a wiki page that explains how updates work for Dangerzone, before
 # merging this.
 MSG_CONFIRM_UPDATE_CHECKS = """\
-Do you want to be notified about new Dangerzone releases?
+<p>Do you want to be notified about new Dangerzone releases?</p>
 
-If "Yes", Dangerzone will check GitHub for new releases on startup. If "No", Dangerzone will make no network requests and won't inform you about new releases.
+<p>If <i>"Yes"</i>, Dangerzone will check GitHub for new releases on startup. If <i>"No"</i>, Dangerzone will make no network requests and won't inform you about new releases.</p>
 
-If you prefer another way of getting notified about new releases, we suggest adding to your RSS reader our Mastodon feed: https://fosstodon.org/@dangerzone.rss. For more information about updates, check our wiki page: https://github.com/freedomofpress/dangerzone/wiki/Updates
+<p>If you prefer another way of getting notified about new releases, we suggest adding to your RSS reader our <a href="https://fosstodon.org/@dangerzone.rss">Mastodon feed</a>. For more information about updates, check our <a
```

Comment:
I don't think it matters much for the html parser but perhaps the start of `<a>` should be at the beginning of the new line if having this line break here feels needed at all.

Reply:
Actually, the whole "long lines" thing was an artifact from the time that our dialog messages were not formatted in HTML. Qt internally relied on line breaks to understand where each text paragraph should go. Now that the messages are formatted with `<p>...</p>`, this is no longer a necessity, and we can break the text any way we want.

So, to improve readability, I've wrapped the text at 88 chars (what Black does internally).